.gitignore

```diff
@@ -11,3 +11,4 @@
 .rspec
 /kubernetes/tmp
 /kubernetes/manifests
+/kubernetes/salt/config/minion.d/_schedule.conf
```

Comment:
this is automatically generated on my machine too. Maybe we need to gitignore it?